notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>What do you think about changing plotting options below to show some underlying scatter structure if any?</p><ul><li>smaller marker size</li><li>darker edgecolor</li><li>a bit transparent color so use alpha &lt; 1</li></ul><p>Please do the same application to the next CMD.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I like how you updated the plot. And yet, a standard CMD plot shows underlying physical characteristics of the system but this doesn't. It's not because of the plotting style but because we haven't applied more filters, I think. Though at the moment, I am not fully sure what further criteria is needed to make this CMD look more like a standard CMD which shows its evolutionary track of the system. However, we can try a few things.</p><p> </p><p>So I suggest</p><p>	    </p><ol><li>Calculate distance modulus ( m-M ) of the system and the distance in this field. Leo I has m-M = 21.8 and distance = 254 Kpc.</li><li>option: I wonder if you can calculate distance of each star from the center of the galaxy.  If you can, you can apply another filter of averaged half-ligh radius (226 pc) to the data. </li><li>If you look at Section 5.1, you will see some more criteria for PS1 for old, metal-poor stars with (16 &lt;= m-M &lt;=22). Overplot these data within that distance modulus range (and half-light radius) with the current plotting style.</li><li>Make sure to overplot the original data before the above critera with simple very small dots with light grey (no edgecolor but much smaller size)</li><li>Follow the above steps for other CMD plots but make sure the other galaxy has a different distance modulus and so on.</li></ol><p> </p><p> </p>

Reply:
<p>For the record, I followed this paper: https://iopscience.iop.org/article/10.3847/1538-4357/aac16b, referenced by Drelica-Wagner for Leo I to make the CMD. I think the one I have is in good agreement with their Figure 20. </p><p>I made the 4.5' filter and it did change the CMD a bit but the distance modulus filter did not do anything. I can leave the code in their if you would like to change it later. </p>

Reply:
<p>The small grey points for the outer stars doesn't tell us much but I'll leave it. I have run out of time to make the more complicated changes here. my apologies. </p>